app/backend/graphql/customer/mutations/customerDeleteMutationField.ts

```diff
@@ -0,0 +1,16 @@
+import { idArg, mutationField } from 'nexus'
+import { isTeamMember } from '../../team/utils'
+import { Customer } from '../customer'
+
+export const customerDeleteMutationField = mutationField('customerDelete', {
+  type: Customer,
+  description: 'Delete a customer',
+  args: {
+    customerId: idArg({ description: 'Id of the customer' }),
+  },
+  authorize: async (_source, { customerId }, context) => {
+    const customer = await context.prisma.customer.findUnique({ where: { id: customerId }, rejectOnNotFound: true })
+    return isTeamMember({ id: customer.teamId }, context)
+  },
+  resolve: (_source, { customerId }, context) => context.prisma.customer.delete({ where: { id: customerId } }),
```

Comment:
We need to be sure that no project with time- or invoice data gets deleted accidentally

Reply:
Let's look at that again when we link projects and invoices to the customer.